common/setups/rasr/gmm_system.py

```diff
@@ -1237,6 +1244,23 @@ def run_vtln_sat_step(self, step_args):
                     **step_args.sdm_args,
                 )
 
+    def run_forced_align_step(self, step_args):
```

Comment:
I wonder if it would be cleaner to add this as `_v2` and the old one as `run_forced_align_step`. Otherwise it is extremely unclear why it does not match.